CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
+
+### Changed
+* The integration by default does not report exited containers that are older than 24 hours.
```

Comment:
I have a slight preference to state that this behavior actually changed, even if it is already in the "changelog section". Feel free to touch up the wording as you like though.
```suggestion
* Since 1.3.4, nri-docker will not report containers that have been stopped for more than 24 hours.
```

Reply:
I didn't do this because I'm not sure next release will be `1.3.4`. It could be `1.4.0` because there's a new feature. Or you could consider it a breaking change and do `2.0.0`, or other things could be merged and then it's impossible to predict what's the next release version.

Reply:
Agreed, `1.3.4` should better be a placeholder. I'll edit the sugestion.